sdk/spring/spring-cloud-azure-appconfiguration-config/pom.xml

```diff
@@ -142,12 +137,12 @@
           <rules>
             <bannedDependencies>
               <includes>
+				<include>jakarta.annotation:jakarta.annotation-api:[2.1.1]</include> <!-- {x-include-update;jakarta.annotation:jakarta.annotation-api;external_dependency} -->
```

Comment:
Seems like the only annotations we are using is `PostConstruct`. I'd recommned that we use `implements InitializingBean` instead, so we don't need to depend on the `jakarta.annotations` directly.